docs/current_docs/quickstart/snippets/publish/go/dagger.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "my-module",
```

Comment:
This and other quickstart `dagger.json` module names should be `hello-dagger`

Reply:
Nice catch - putting up a PR soon! :pray: